JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -202,6 +209,17 @@ - (void)jsq_configureMessagesViewController
 
     self.topContentAdditionalInset = 0.0f;
 
+    self.bottomContentAdditionalInset = 0.0f;
+
+    self.isAdVisible = YES;
+
+//    [self setInputToolbarShadowVisible:YES];
+
+    self.adContainerView.clipsToBounds = YES;
```

Comment:
is this duplicative of the code below? Let's just have in one place if possible.